client/pages/forgot.js

```diff
@@ -0,0 +1,81 @@
+import {
+  Button,
+  FormControl,
+  Flex,
+  Heading,
+  Input,
+  Stack,
+  Text,
+  useColorModeValue,
+} from "@chakra-ui/react";
+import { useState } from "react";
+
+const ResetPassword = () => {
+  const [resetPasswordInput] = useState({
```

Comment:
Rename this to `forgotPasswordInput` and `setForgotPasswordInput` just this this 
```jsx
const [forgotPasswordInput, setForgotPasswordInput] = useState({ email: "" });
```

useState return 2 type the first one the data that is `forgotPasswordInput` and the second one this `setForgotPasswordInput` that is a function that is used to update `forgotPasswordInput`